lstchain/mc/tests/test_mc.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from numpy.testing import assert_allclose
+from lstchain.mc import power_law_integrated_distribution
+
+def test_integrated_distribution():
+
+    emin = 50.     # u.GeV
+    emax = 500.e3  # u.GeV
+    Nevents = 1e6
+    spectral_index = -2
```

Comment:
Note that this test would actually pass with the previous version of the code, because `spectral_index = -2` is a special case.
I would suggest testing with another value (not integer).

Reply:
Thanks for the review @vuillaut. I took into account your comments in the PR

Reply:
Well, thank you for the fix.
Good to merge.